### PR TITLE
Wipe data button added to clear all existing data

### PR DESCRIPTION
**Pull request by Bismay5467**
Resolves issue #291 

@Teddarific @alexdanilowicz 
Can you please review this PR?
Any feedback is always appreciated.

### CODE REVIEW

File: packages/core/src/components/Layout.tsx [hunk 1]
```diff
@@ -120,6 +152,12 @@ export default function Layout({
           onClose={onExportSuccessModalClose}
           primitives={{ colors, typography, shadows }}
         />
+        <AlertDialogDelete
+          isOpen={isAlertDialogOpen}
+          onClose={onDeleteAlertDialogClose}
+          onDelete={() => onDeleteData()}
+          deleteAllTokens={true}
+        />
```
Comment (alexdanilowicz) [inline packages/core/src/components/Layout.tsx:160]: can you mimic the Export Settings Section and have it be in it's own "Settings"section? 

Screen Shot 2023-04-11 at 11 28 14 PM

**Review by alexdanilowicz — COMMENTED** [review]
Nice. thanks for this. a few comments.

Also @Teddarific what do you think about this... i feel like the delete all tokens should only be on web? It doesn't make much sense to me in the package...

I think this should only show on web.
```
  {platform === 'web' && (    <AlertDialogDelete
          isOpen={isAlertDialogOpen}
          onClose={onDeleteAlertDialogClose}
          onDelete={() => onDeleteData()}
          deleteAllTokens={true}
        />)}
```

**Comment by Teddarific** [comment]
I think not showing it on package makes sense to me. Don't feel strongly either way.

RE: UI, it looks like it's getting a bit cluttered — my proposal is something like this:

image


The changes being:

- Under export, re name that "Settings" link to "File Types" and switch around the icons
- We add a settings icon in the bottom left, that opens a Modal with the clear all option

**Comment by alexdanilowicz** [comment]
@Bismay5467 are you able to make the changes @Teddarific proposed above? No rush. Thank you so much! 🎊 Appreciate all your help, and hope examinations at your university are going well.

File: packages/mirrorful/editor/package.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
     "eslint": "^8.36.0",
     "eslint-config-next": "13.1.6",
     "framer-motion": "^10.6.0",
-    "next": "^13.3.0",
```
Comment (alexdanilowicz) [inline packages/mirrorful/editor/package.json]: Can we switch this back to next 13.3 @Bismay5467

**Comment by alexdanilowicz** [comment]
Moving conversation to here: https://mirrorful.slack.com/archives/C04RR2D7C65/p1681607497675869?thread_ts=1681281279.665279&cid=C04RR2D7C65 we can do a fast follow. For now, looks great, merging this in!

**Pull request merged**